test/correctness/simd_op_check.cpp

```diff
@@ -232,7 +231,7 @@ class SimdOpCheck : public SimdOpCheckTest {
 
         if (use_ssse3) {
             for (int w = 2; w <= 4; w++) {
-                check("pmulhrsw", 4 * w, i16((((i32(i16_1) * i32(i16_2)) + 16384)) / 32768));
+                check("pmulhrsw", 4 * w, i16((i32(i16_1) * i32(i16_2) + 16384) >> 15));
```

Comment:
...I am genuinely curious as to how you noticed this; do you have a non-sse3 machine somewhere?

Reply:
I think before this PR (or my last one), we were incorrectly generating pmulhrsw on non-sse3 targets, and there was a simd_op_check for that. When I fixed that (by accident due to refactoring/cleanup), simd_op_check started failing because it was expecting pmulhrsw when it shouldn't be generated.